STX.REST.RESTFulSense.Clients/Brokers/ErrorBrokers/ErrorBroker.StatusDetails.cs

```diff
@@ -0,0 +1,893 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using STX.REST.RESTFulSense.Clients.Models.Brokers.StatusDetails;
+
+namespace STX.REST.RESTFulSense.Clients.Brokers.ErrorBrokers
```

Comment:
The namespace ErrorBrokers I think it should be "STX.REST.RESTFulSense.Clients.Brokers.Errors".